src/utils/runtime/src/logger.rs

```diff
@@ -492,6 +510,18 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
             .with_filter(reload_filter);
 
         layers.push(layer.boxed());
+
+        let reporter = OpenTelemetryReporter::new(
+            exporter,
+            SpanKind::Server,
+            Cow::Owned(Resource::new([KeyValue::new(
```

Comment:
The resource can be reused as well.

Reply:
The service name should be different from the distributed tracing.

Reply:
Interesting. Is it because there is currently no way to maintain the parent relationship between tracing spans and minitrace spans? So we'd rather make them not interleaving with each other by assigning a different service name.

Reply:
Emmm, nop. It is because I think it is not a good idea to assign the distributed tracing and lib-level tracing in the same service name. 👀 Separating them can give a clearer view.